corehq/motech/repeaters/models.py

```diff
@@ -244,6 +244,44 @@ def reset_next_attempt(self):
             self.save()
 
 
+class SQLCaseRepeater(SQLRepeater):
+    version = models.CharField(
+        max_length=10,
+        choices=list(zip(LEGAL_VERSIONS, LEGAL_VERSIONS)),
+        default=V2,
+    )
+
+    # An empty value means all case types are accepted
+    white_listed_case_types_str = models.TextField(default='')
```

Comment:
I noticed that we use `django.contrib.postgres.fields.ArrayField` in some places in HQ. This might be an easier fit here, and avoid the need for the properties below.